bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -88,6 +92,11 @@
      */
     public static final String PN_MAIN_CONTENT_SELECTOR_PROP = "mainContentSelector";
 
+    /**
+     * Property name of the style property that enables/disables rendering of the alternate language links
+     */
+    public static final String PN_STYLE_RENDER_ALTERNATE_LANGUAGES = "renderAlternateLanguageLinks";
```

Comment:
I'd call it `PN_STYLE_RENDER_ALTERNATE_LANGUAGE_LINKS`, in sync with the repository property.